test/e2e/framework.go

```diff
@@ -1651,10 +1642,25 @@ func randSeq(n int) string {
 	return string(b)
 }
 
+func randSequence(n int) string {
+	b := make([]rune, n)
+	for i := range b {
+		randIdx := rand.Intn(len(letters))
+		b[i] = letters[randIdx]
+	}
+	return string(b)
+}
+
+// randName generates a DNS-1123 subdomain name
 func randName(prefix string) string {
 	return prefix + randSeq(nameSuffixLength)
 }
 
+// randNamespace generates the name of a namespace
+func randNamespace(prefix string) string {
```

Comment:
It doesn't seem necessary to have a dedicated function for namespace. `randName` can be used to generate the "name" of namespace.

Reply:
Yes if the above changes are acceptable then I can use the same function randName with different parameters(character array).